src/Agent.Plugins/PipelineArtifact/PipelineArtifactServer.cs

```diff
@@ -24,6 +24,7 @@ public class PipelineArtifactServer
         public static readonly string ProofNodes = "ProofNodes";
         public const string PipelineArtifactTypeName = "PipelineArtifact";
         public const string BuildArtifactTypeName = "Container";
+        public const string ArtifactSize = "artifactsize";
```

Comment:
Use Uppercase s? artifactSize

Reply:
This constant is lowercase on the server side as well, so I wanted to be consistent